mozilla/foundation.mozilla.org
# 8190 - Add datawrapper to style-guide factory

Closes #8190 

Should also include a link to https://foundation.mozilla.org/en/docs/how-do-i-wagtail/datawrapper/

Also let me know if you have any feedback about the how to article I made!

https://foundation-s-8190-dataw-gn23my.mofostaging.net/en/style-guide/#datawrapper

To test: 
* scroll to bottom of style-guide and check the datawrapper component is visible
